src/js/tools/drawing/Lighten.js

```diff
@@ -65,9 +70,15 @@
       }
     }
     if (color) {
-      usedPixels[key] = true;
-      this.superclass.applyToolAt.call(this, col, row, color.toRgbString(), frame, overlay, event);
+      // Convert tinycolor color to string format.
+      color = color.toRgbString();
+    } else {
+      // Not sure why this check exists in the first place.
```

Comment:
After checking the code you can get rid of the check : tinycolor and tinycolor.darken/lighten always return a valid tinycolor instance.